docs/samples/pytorch/README.md

```diff
@@ -80,7 +80,15 @@ $ inferenceservice.serving.kubeflow.org/pytorch-cifar10 created
 MODEL_NAME=pytorch-cifar10
 INPUT_PATH=@./input.json
 CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+For Clusters which do not have Load Balancers the above command wont work due to the null value for .status.loadBalancer from json output.
+Hence The Below Command Needs to be Run in case of Clusters who do not use load balancer and use Node Port to expose:
 
+```
```

Comment:
Good catch! The examples need the same fix, maybe the best way is to wrap this into a common script ?

Reply:
https://deploy-preview-1492--competent-brattain-de2d6d.netlify.com/docs/components/serving/pytorchserving/ look at this page of kubeflow 0.70.
i think for a better user experience this page needs to be updated because 0.70 wen showing a different story and kserving github md tells a different story .I think when i stated implementing the kubeflow for the first as an user it was very confusing one using ksonnet and the other saying something different.This is a case of inconsistency in documentation.Please provide me your views?

Reply:
Sorry for the slow response!

We tend to copy this readme around. A central place is useful, but makes it hard to find. I think adding it here, as well as the other framework examples is a great start.

Consider doing something like 
```
# If calling from outside the cluster using a loadbalancer
IP_JSON_PATH="{.status.loadBalancer.ingress[0].ip}"
# If calling from inside the cluster
IP_JSON_PATH="{.spec.clusterIP}" 
```

Also, do we want to use status.clusterIP instead of spec?